return external_call(global._sdl_error)
